src/main/java/com/vinylteam/vinyl/service/UserPostService.java

```diff
@@ -0,0 +1,9 @@
+package com.vinylteam.vinyl.service;
+
+import com.vinylteam.vinyl.entity.UserPost;
+
+public interface UserPostService {
+    boolean processAdd(UserPost post);
+
+    boolean add(UserPost post);
```

Comment:
For future. Please add empty row after interface declaration and before closing it. It is kind of our code style